src/python/pants/notes/master.rst

```diff
@@ -4,6 +4,96 @@ Master Pre-Releases
 This document describes development releases which occur weekly from master, and which have
 not undergone the vetting associated with ``stable`` releases.
 
+1.26.0.dev0 (02/11/2020)
+------------------------
+
+API Changes
+~~~~~~~~~~~
+
+* Revert "Switch to dot idea project type (#8745)" (#9096)
+  `PR #9096 <https://github.com/pantsbuild/pants/pull/9096>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Add support for TOML config files (`pants.toml`) (#9052)
+  `PR #9052 <https://github.com/pantsbuild/pants/pull/9052>`_
+
+* Add `migrate_to_toml_config.py` script to automatically update INI config files to TOML (#9054)
+  `PR #9054 <https://github.com/pantsbuild/pants/pull/9054>`_
+
+* Add `--run-args` to allow passing arguments in V2 (#9075)
+  `PR #9075 <https://github.com/pantsbuild/pants/pull/9075>`_
+
+Bugfixes
+~~~~~~~~
+
+* Re-pin pytest, and add a pin for zipp. (#9095)
+  `PR #9095 <https://github.com/pantsbuild/pants/pull/9095>`_
+
+* remove hardcoded toolchain variant and fix fingerprinting (#9085)
+  `PR #9085 <https://github.com/pantsbuild/pants/pull/9085>`_
+
+* Allow test specs to be used with --no-fast for junit (#9093)
+  `PR #9093 <https://github.com/pantsbuild/pants/pull/9093>`_
+
+* Fix deprecation warning for `--transitive` for goals that ignore the option (#9076)
+  `PR #9076 <https://github.com/pantsbuild/pants/pull/9076>`_
+
+Refactoring, Improvements, and Tooling
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Nicer rule graph viz (#7024)
+  `PR #7024 <https://github.com/pantsbuild/pants/pull/7024>`_
+
+* Handle Ctrl-C in the V2 UI (#9043)
+  `PR #9043 <https://github.com/pantsbuild/pants/pull/9043>`_
+
+* expose a v2 ruleset for BinaryToolBase (#8859)
+  `PR #8859 <https://github.com/pantsbuild/pants/pull/8859>`_
```

Comment:
@cosmicexplorer is this fine to be in refactoring or you think should be a new feature? Generally, we've been classifying changes to the V2 API—such as renaming `ProvenancedBuildFileAddress` to `AddressWithOrigin`—as refactoring. This is a very nice new feature for rule authors, but not a new feature for your Everyday Pants User™️